src/controllers/ioSignController.ts

```diff
@@ -264,7 +265,30 @@ export default class IoSignController {
       )()
     );
 
-  public readonly getQtspClausesMetadata = (): Promise<
-    IResponseErrorInternal | IResponseSuccessJson<QtspClausesMetadataDetailView>
-  > => this.ioSignService.getQtspClausesMetadata();
+  public readonly getQtspClausesMetadata = (
+    req: express.Request
+  ): Promise<
+    | IResponseErrorInternal
+    | IResponseErrorValidation
+    | IResponseSuccessJson<QtspClausesMetadataDetailView>
+  > =>
+    withUserFromRequest(req, async () =>
```

Comment:
Do we want this endpoint to be authenticated now?

Reply:
If I delete `withUserFromRequest` using `withValidatedOrValidationError` I get this type error:

` Type 'IResponse<"IResponseErrorValidation">' is missing the following properties from type 'Promise<IResponseErrorValidation | IResponseErrorInternal | IResponseSuccessJson<....`

Reply:
Ok I deleted `withUserFromRequest` and added `withCatchAsInternalError`

Reply:
can you check that all io-sign endpoints are wrapped with `withCatchAsInternalError` (or at least that all exceptions are caught) ? consider that uncaught exceptions may kill the whole backend application.

Reply:
all other endpoints use `withUserFromRequest` which internally uses `withCatchAsInternalError` so they are all already protected!